skactiveml/utils/_selection.py

```diff
@@ -140,3 +143,80 @@ def simple_batch(
         return best_indices, batch_utilities
     else:
         return best_indices
+
+
+def combine_ranking(*iter_ranking, rank_method=None, rank_per_batch=False):
```

Comment:
Maybe move to representativeness and diversirty? Or do you think that this method will be used by others, e.g., for strategy combination?

Reply:
Please comment @LukasLuehrs